SR2023/2023-01-19-competition-venue-announcement.md

```diff
@@ -0,0 +1,35 @@
+---
+to: Student Robotics 2023 teams
+subject: SR2023 Competition Venue
+---
+
+Hello!
+
+We're thrilled to announce the venue for Student Robotics 2023!
+
+The competition will be held from *Saturday 1st April* until *Sunday 2nd April*, at the University of Southampton, so please plan accommodation / transport accordingly. The exact start/end times are to be confirmed, but doors open 9am, and we aim to finish around 6pm.
```

Comment:
The website says aim to finish around 5pm, not 6pm, which is right?

Reply:
https://github.com/srobo/team-emails/pull/136/commits/d33ec07f3b324b2ec636b01f8a62c78cc3944960